bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -139,13 +140,16 @@ public void rewind(final int baseSeconds, final int windowId) throws IOException
     }
 
     /*
-      TODO: properly rewind Data List iterators, especially handle case when iterators point to blocks past the last block.
-    */
+     * TODO: properly rewind Data List iterators, especially handle case when iterators point to blocks past the last
+     *  block.
+     */
 
     final int numberOfInMemBlockPermits = this.numberOfInMemBlockPermits.addAndGet(numberOfInMemBlockRewound);
-    assert numberOfInMemBlockPermits < MAX_COUNT_OF_INMEM_BLOCKS : "Number of in memory block permits " + numberOfInMemBlockPermits + " exceeded configured maximum " + MAX_COUNT_OF_INMEM_BLOCKS + '.';
+    assert numberOfInMemBlockPermits < MAX_COUNT_OF_INMEM_BLOCKS : "Number of in memory block permits " +
+        numberOfInMemBlockPermits + " exceeded configured maximum " + MAX_COUNT_OF_INMEM_BLOCKS + '.';
     resumeSuspendedClients(numberOfInMemBlockPermits);
-    logger.debug("Discarded {} in memory blocks during rewind. Number of in memory blocks permits {} after rewinding {}. ", numberOfInMemBlockRewound, numberOfInMemBlockPermits, this);
+    logger.debug("Discarded {} in memory blocks during rewind. Number of in memory blocks permits {} after" +
+        " rewinding {}.", numberOfInMemBlockRewound, numberOfInMemBlockPermits, this);
```

Comment:
I disagree that we should allow exceptions to code style rules. Either 120 line length needs to be enforced or it should not be enforced at all, not just for string literals. Note that some string literals may be very long and span multiple lines.